test/bzz-protocol.spec.ts

```diff
@@ -172,23 +216,28 @@ describe('BZZ protocol', () => {
     done()
   })
 
+  afterAll(async done => {
+    await saveFile(rootIndexPath, originalIndex)
+    done()
+  })
```

Comment:
This returns the file into original state.

Reply:
but it does not work on the file itself, it requests on the file already uploaded to Swarm

Reply:
It updates the `test/bzz-test-page/index.html`, but it actually doesn't have to return it back to the original state on the end of the tests. Because if the tests are interrupted, the file won't be reverted. So this should be moved to the beforeAll, after the page has been uploaded.